### PR TITLE
Added new backend test.

### PR DESCRIPTION
**Pull request by sanket10078**

### CODE REVIEW

File: back-end/test-2/.ddev/config.yaml [hunk 1]
```diff
@@ -0,0 +1,260 @@
+name: back-end
```
Comment (babita-neog) [inline back-end/test-2/.ddev/config.yaml]: @sanket10078  Change the project name to something more meaningful as it is being repeated in both test-1 & test-2.

File: back-end/test-2/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+We use Docker Desktop and DDEV for Drupal our local development. If you prefer a different local development enviroment, please use what you are comfortable with or already have set up.
+
+To install Docker and DDEV please follow their instructions here: 
+* [Docker](https://ddev.readthedocs.io/en/latest/users/install/docker-installation/)
+* [DDEV](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/)
+
+Once DDEV and Docker are set up you can start the project by running the following commands:
+
+```
+cd back-end
```
Comment (babita-neog) [inline back-end/test-2/README.md]: @sanket10078 update the instructions as now path is changed to back-end/test-2

File: back-end/test-2/.ddev/config.yaml (hunk 1 above)
Comment (sanket10078) [reply back-end/test-2/.ddev/config.yaml]: updated

File: back-end/test-2/README.md (hunk 2 above)
Comment (sanket10078) [reply back-end/test-2/README.md]: updated

**Comment by sanket10078** [comment]
@babita-neog I have tested the steps mentioned in the doc on my local and it works fine. 
Screenshot 2024-08-22 at 7 34 22 PM

File: back-end/test-1/README.md
Comment (babita-neog) [inline back-end/test-1/README.md:1]: @sanket10078 please also update the steps here.

Screenshot 2024-08-22 at 8 08 45 PM

Comment (sanket10078) [reply back-end/test-1/README.md:1]: updated

**Review by babita-neog — APPROVED** [review]

**Pull request merged**